cpp/src/arrow/util/rle_encoding_test.cc

```diff
@@ -33,6 +33,8 @@
 #include "arrow/util/io_util.h"
 #include "arrow/util/rle_encoding.h"
 
+#include <stdio.h>
```

Comment:
Could you revert needles changes such as this `include` and other empty line changes?